_translations/en.yml

```diff
@@ -54,27 +50,21 @@ en:
+    owntext: "If you don't use any third party APIs, you can integrate a Bitcoin server directly in your applications, allowing you to become your own bank and payment processor. With all the responsibilities that this implies, you can build amazing systems that process Bitcoin transactions however you would like."
     invoice: "Bitcoin addresses to track invoices"
     invoicetext: "Bitcoin creates a unique address for each transaction. So if you were to build a payment system associated with an invoice, you simply need to generate and monitor a Bitcoin address for each payment. You should never use the same address for more than one transaction."
     security: "Most of the security is on client side"
-    securitytext: "Most parts of the security are handled by the protocol. This means no need for PCI compliance and fraud detection is only required when services or products are delivered instantly. Storing your bitcoins in a <a href=\"#secure-your-wallet#\">secure environment</a> and securing payment requests displayed to the user should be your main concerns."
-    micro: "Cheap micro payments"
-    microtext: "Bitcoin offers the lowest payment processing fees and usually can be used to send micro payments as low as a few dollars in value. Bitcoin allows to design new creative online services that could not exist before only because of financial limitations. This includes various kinds of tipping systems and automated payment solutions."
+    securitytext: "Most parts of the security are handled by the protocol. This means no need for PCI compliance, and fraud prevention can be simplified down to monitoring a single variable: the confirmation score. Storing your bitcoins in a <a href=\"#secure-your-wallet#\">secure environment</a> and securing payment requests displayed to the user should be your main concerns."
   bitcoin-for-individuals:
     title: "Bitcoin for Individuals - Bitcoin"
     pagetitle: "Bitcoin for Individuals"
     summary: "Bitcoin is the simplest way to exchange money at very low cost."
     mobile: "Mobile payments made easy"
     mobiletext: "Bitcoin on mobiles allows you to pay with a simple two step scan-and-pay. No need to sign up, swipe your card, type a PIN, or sign anything. All you need to receive Bitcoin payments is to display the QR code in your Bitcoin wallet app and let your friend scan your mobile, or touch the two phones together (using NFC radio technology)."
-    international: "Fast international payments"
-    internationaltext: "Bitcoins can be transferred from Africa to Canada in 10 minutes. There is no bank to slow down the process, level outrageous fees, or freeze the transfer. You can pay your neighbors the same way as you can pay a member of your family in another country."
```

Comment:
Luke, you argued earlier that we should keep "instant peer-to-peer transactions" above because 0-conf is feasible among trusted friends / family.  The line in question here also implies sending to friends / family abroad, so we should actually change it from "in 10 minutes" to "instantly" here as well, rather than "hours".